test/karma/firefox.js

```diff
@@ -0,0 +1,8 @@
+var common = require('./common');
+
+module.exports = function(config) {
+  config.set(common({
+    plugins: ['karma-firefox-launcher'],
+    browsers: ['Firefox']
+  }));
+};
```

Comment:
I don't think running one specific browser at a time means we need separate files for each of the configs but that sounds like it's a generator problem, not an issue with this patch.